code/go/0chain.net/smartcontract/storagesc/handler.go

```diff
@@ -1513,17 +1513,27 @@ func (srh *StorageRestHandler) getWriteMarker(w http.ResponseWriter, r *http.Req
 //      description: restrict to transactions in indicated block
 //      in: query
 //      type: string
+//    + name: start_block_number
+//      description: restrict to transactions in specified start block and endblock
+//      in: query
+//      type: string
+//    + name: end_block_number
```

Comment:
let's keep the query params simple and use hyphens, so`block-start` and `block-end` should be fine

Reply:
Yes fixed, I should change it in other PRs too, this is definitely better and simple :). Thanks!